README.md

```diff
@@ -63,6 +63,11 @@ JENKINS_USER=user
 JENKINS_TOKEN=password
 ```
 
+#### TeamCity
+
+    TC_USER=user
+    TC_PASSWORD=password
```

Comment:
I'd prefer it if we used more explicit variable names please, e.g. `TEAM_CITY_USER`.

Reply:
Agree

Reply:
fixed